vyper/parser/parser_utils.py

```diff
@@ -531,8 +527,7 @@ def make_setter(left, right, location, pos, in_function_call=False):
 
         left_token = LLLnode.from_list("_L", typ=left.typ, location=left.location)
         if left.location == "storage":
-            left = LLLnode.from_list(["sha3_32", left], typ=left.typ, location="storage_prehashed")
-            left_token.location = "storage_prehashed"
+            pass
```

Comment:
Can you add a comment why `pass` is the appropriate path to take here? Can it be skipped entirely?

Reply:
Ahh yes this is a mistake.  I did that in a few places while I was working, thought I'd removed them all before opening PR.